parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -415,33 +431,35 @@ try
             -IdentitySystem "{{ GetIdentitySystem }}"
     }
 
+    Logs-To-Event -TaskName "AKS.WindowsCSE.NewExternalHnsNetwork" -TaskMessage "Start to create new external hns network"
     New-ExternalHnsNetwork -IsDualStackEnabled $global:IsDualStackEnabled
-
+    
+    Logs-To-Event -TaskName "AKS.WindowsCSE.InstallKubernetesServices" -TaskMessage "Start to install kubernetes services"
     Install-KubernetesServices `

```

Comment:
Maybe we can log these event inside the function. This can reduce the length of customdata